app/Policies/MessagePolicy.php

```diff
@@ -46,4 +48,25 @@ public function forceDelete(User $user, Message $message): bool
     {
         return false;
     }
+
+    public function sendToRecipient(User $user, User $targetUser): bool
+    {
+        $isUserDeveloper = $user->hasAnyRole([
+            Role::DEVELOPER,
+            Role::DEVELOPER_JUNIOR,
+            Role::DEVELOPER_STAFF,
+        ])
+            || $user->getAttribute('Permissions') >= Permissions::JuniorDeveloper;
```

Comment:
RACheats, WritingTeam, RANews, and TheUnwanted don't have developer permissions, so won't be able to reply. We probably want to attribute all of the shared accounts with a "POINT_OF_CONTACT" role and add that to this list.

Reply:
Added a new role in latest:
![Screenshot 2024-06-07 at 6 41 01 PM](https://github.com/RetroAchievements/RAWeb/assets/3984985/cc932197-637f-4ef9-abb4-c912454cdd3a)

Also updated the PR description with some SQL needed to add it to the DB. After deployment, I can attach the role to:
* RAdmin
* RACheats
* DevCompliance
* QATeam
* WritingTeam
* RANews
* RAEvents
* TheUnwanted
* DevQuest
* RAArtTeam